src/abstractdataframe/io.jl

```diff
@@ -240,14 +240,14 @@ Data.streamfrom(source::DataFrame, ::Type{Data.Field}, ::Type{T}, row, col) wher
 Data.streamtypes(::Type{DataFrame}) = [Data.Column, Data.Field]
 Data.weakrefstrings(::Type{DataFrame}) = true
 
-allocate(::Type{T}, rows, ref) where {T} = Vector{T}(uninitialized, rows)
-allocate(::Type{CategoricalString{R}}, rows, ref) where {R} = CategoricalArray{String, 1, R}(uninitialized, rows)
+allocate(::Type{T}, rows, ref) where {T} = Vector{T}(undef, rows)
+allocate(::Type{CategoricalString{R}}, rows, ref) where {R} = CategoricalArray{String, 1, R}(undef, rows)
```

Comment:
I'm surprised this didn't require your update to CategoricalArrays in order to pass tests.

Reply:
Nice